hooks/useCustomToasts.ts

```diff
@@ -33,7 +33,7 @@ const ERROR_MESSAGES = {
   "Token Array Empty": "No tokens provided for sale.",
   "Unable to decode signature":
     "Please run 'mintNextBatch' first before creating a sale",
-  "Size of bytes": "Invalid wallet address(es) submitted",
+  "Size of bytes": "Invalid merkle root submitted",
```

Comment:
@EsterB12 I am not sure if the client would understand this. @CourtneyLinum what do you think?